frontend/src/store/modules/website/actions.js

```diff
@@ -8,38 +12,8 @@ export default {
     [SAVE_NEW_WEBSITE]: (context) => {
         const newDataSite = context.state.newWebsite;
 
-        return new Promise((resolve, reject) => {
-            if(!newDataSite.name || !newDataSite.domain) {
-                reject({
-                    errors: {
-                        message: "Fill in all thi fields correctly."
-                    }
-                });
-            }
-            const fakeWebsiteData = {
-                name: 'Website name',
-                domain: 'http://domain.com',
-                single_page: newDataSite.single_page,
-                tracking_number: '123456789-10',
-            };
-
-            if (fakeWebsiteData.tracking_number && fakeWebsiteData.name === newDataSite.name && fakeWebsiteData.domain === newDataSite.domain) {
-                context.commit(ADD_WEBSITE, fakeWebsiteData);
-                resolve(fakeWebsiteData);
-            } else if (fakeWebsiteData.name !== newDataSite.name) {
-                reject({
-                    errors: {
-                        name: "Wrong name of website"
-                    }
-                });
-            } else if (fakeWebsiteData.domain !== newDataSite.domain) {
-                reject({
-                    errors: {
-                        domain: "Wrong domain of website"
-                    }
-                });
-            }
-
-        });
+        return requestService.create(resourceUrl, newDataSite)
```

Comment:
It's better to create API service that will responsible for updating data (see `api/auth.js` for example